src/content/en/fundamentals/instant-and-offline/offline-storage-pwa/index.md

```diff
@@ -0,0 +1,260 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2016-09-29 #}
+{# wf_published_on: 2016-09-29 #}
+
+# **Offline Storage for Progressive Web Apps** {: .page-title }
+
+{% include "web/_shared/contributors/addyosmani.html" %}
+
+![image alt text](images/image_0.jpg)
+
+The Pokedex.org Progressive Web App uses IndexedDB for application state and
+the Pokemon data set while the Cache API is used for URL addressable resources.
+
+Internet connections can be flakey or non-existent on the go, which is why
+offline support and reliable performance are common features in [Progressive
+Web Apps](https://developers.google.com/web/progressive-web-apps/). Even in
```

Comment:
@petele would note that we should prolly be linking to /web/progressive-web-apps/ here